### PR TITLE
Fixing iteration count and kmin,kmax for ecn scripts

### PR DESCRIPTION
**Pull request by selldinesh**
### Description of PR

Summary: Fixing iteration count getting overwritten by the helper file and kmin, kmax values for ecn scripts inorder to support all platform types
Fixes # (issue) 
https://github.com/sonic-net/sonic-mgmt/issues/12852

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311

#### What is the motivation for this PR?
fix for Fixing iteration count getting overwritten
#### How did you do it?
Removed the line that is responsible for overwriting the iteration count and modofoed the kmin and kmax value in order to support all platform types

### CODE REVIEW

**Comment by selldinesh** [comment]
Hi @developfast ,
Request to approve this PR

File: tests/snappi_tests/ecn/test_red_accuracy_with_snappi.py [hunk 1]
```diff
@@ -63,10 +63,10 @@ def test_red_accuracy(request,
     snappi_extra_params = SnappiTestParams()
     snappi_extra_params.packet_capture_type = packet_capture.IP_CAPTURE
     snappi_extra_params.is_snappi_ingress_port_cap = True
-    snappi_extra_params.ecn_params = {'kmin': 500000, 'kmax': 2000000, 'pmax': 5}
+    snappi_extra_params.ecn_params = {'kmin': 500000, 'kmax': 900000, 'pmax': 5}
```
[the remark this answers is not in the document]
Comment (selldinesh) [reply tests/snappi_tests/ecn/test_red_accuracy_with_snappi.py:66]: it seems this value supports for all card types

File: tests/snappi_tests/multidut/ecn/test_multidut_red_accuracy_with_snappi.py [hunk 2]
```diff
@@ -90,9 +90,9 @@ def test_red_accuracy(request,
 
     snappi_extra_params.packet_capture_type = packet_capture.IP_CAPTURE
     snappi_extra_params.is_snappi_ingress_port_cap = True
-    snappi_extra_params.ecn_params = {'kmin': 500000, 'kmax': 2000000, 'pmax': 5}
+    snappi_extra_params.ecn_params = {'kmin': 500000, 'kmax': 900000, 'pmax': 5}
```
Comment (sdszhang) [inline tests/snappi_tests/multidut/ecn/test_multidut_red_accuracy_with_snappi.py:93]: pls create a seperate PR for this change, not related iteration.

File: tests/snappi_tests/ecn/test_red_accuracy_with_snappi.py (hunk 1 above)
Comment (sdszhang) [inline tests/snappi_tests/ecn/test_red_accuracy_with_snappi.py:66]: pls seperate this

Comment (abdosi) [reply tests/snappi_tests/ecn/test_red_accuracy_with_snappi.py:66]: @selldinesh Please take care of this comment

File: tests/snappi_tests/multidut/ecn/test_multidut_red_accuracy_with_snappi.py (hunk 2 above)
Comment (abdosi) [reply tests/snappi_tests/multidut/ecn/test_multidut_red_accuracy_with_snappi.py:93]: @selldinesh :please take care of this comment so we can approve/merge this asap

Comment (selldinesh) [reply tests/snappi_tests/multidut/ecn/test_multidut_red_accuracy_with_snappi.py:92]: @abdosi I have removed the bgp part and modified the title

**Review by sdszhang — APPROVED** [review]

**Review by abdosi — APPROVED** [review]

**Pull request merged**